gcdu/commands/import_cmd.py

```diff
@@ -61,15 +61,19 @@ def execute_import(project, namespace, data_dir, project_placeholder,
                                                namespace)
     entities_replaced = json.loads(entities_replaced_json)
 
-    inserts = [
-        {'insert': entity} for entity in entities_replaced
-    ]
+    
+    _iter = chunk(entities_replaced, 500)
```

Comment:
What do you think of having the chunk size as an optional command line parameter, with a default value?